app/models/cafe_work.rb

```diff
@@ -33,17 +33,7 @@ def send_email
     Rails.logger.info 'Mailer could not connect, rescued here'
   end
 
-  def old_profile
-    if profile_id.present?
-      if user_id == nil
-        return User.find_by(id: profile_id)
-      else
-        return user
-      end
-    else
-      return user
-    end
-  end
+  alias old_profile user
```

Comment:
LGTM
I do not think this code is used somewhere else, I probably had some idea to use this during the merge Profile -> User.
Perhaps keep it as an alias for a while.